po/it.po

```diff
@@ -57,7 +57,7 @@ msgstr "Preferenze"
 #: src/Widgets/Headerbar.vala:46
 #, fuzzy
 msgid "Show an indicator:"
-msgstr "Mostra l'icona del pannello"
+msgstr "Mostra icona nel pannello"
```

Comment:
I don't understand Italian so it's OK if this is intentional, but the colon at the end of the original string seems to be missing in the translated string.